domains/roleoperations.go

```diff
@@ -42,7 +42,7 @@ const (
 	readPermission            = "read_permission"
 	deletePermission          = "delete_permission"
 	manageRolePermission      = "manage_role_permission"
-	addRoleUsersPermission    = "add_role_users_permission"
+	AddRoleUsersPermission    = "add_role_users_permission"
```

Comment:
Does this operations is needed some where else ? Why it is exported ?

Reply:
here 

```
if err := am.extAuthorize(ctx, auth.EncodeDomainUserID(domainID, inv.InvitedBy), domains.AddRoleUsersPermission, policies.DomainType, domainID); err == nil {
		return err
	}
	```